Run: 8
/cm/local/apps/slurm/var/spool/job995767/slurm_script: line 26: 
pip install -e tpot2
pip install -r tpot2/ImputerExperiments/requirements_.txt
: No such file or directory
RunStart
Version 0.1.7a0 of tpot2 is outdated. Version 0.1.8a0 was released Tuesday September 17, 2024.
starting loops
../data/r/42712/42712.pkl
       season  year  month  hour  holiday  weekday  workingday weather   temp  \
0      spring     0      1     0    False        6       False   clear   9.84   
1      spring     0      1     1    False        6       False   clear   9.02   
2      spring     0      1     2    False        6       False   clear   9.02   
3      spring     0      1     3    False        6       False   clear   9.84   
4      spring     0      1     4    False        6       False   clear   9.84   
...       ...   ...    ...   ...      ...      ...         ...     ...    ...   
17374  spring     1     12    19    False        1        True   misty  10.66   
17375  spring     1     12    20    False        1        True   misty  10.66   
17376  spring     1     12    21    False        1        True   clear  10.66   
17377  spring     1     12    22    False        1        True   clear  10.66   
17378  spring     1     12    23    False        1        True   clear  10.66   

       feel_temp  humidity  windspeed  
0         14.395      0.81     0.0000  
1         13.635      0.80     0.0000  
2         13.635      0.80     0.0000  
3         14.395      0.75     0.0000  
4         14.395      0.75     0.0000  
...          ...       ...        ...  
17374     12.880      0.60    11.0014  
17375     12.880      0.60    11.0014  
17376     12.880      0.60    11.0014  
17377     13.635      0.56     8.9981  
17378     13.635      0.65     8.9981  

[17379 rows x 12 columns]
<class 'pandas.core.frame.DataFrame'>
       season  year  month  hour  holiday  weekday  workingday weather   temp  \
3864   summer     0      6    22    False        2        True   clear  23.78   
15798  winter     1     10    11    False        4        True   misty  22.14   
11766  summer     1      5    11    False        4        True   clear  21.32   
12411  summer     1      6     8    False        3        True   clear  22.14   
14882    fall     1      9     7    False        1        True   clear  21.32   
...       ...   ...    ...   ...      ...      ...         ...     ...    ...   
11844  summer     1      5    17    False        0       False   clear  28.70   
15543  winter     1     10    20    False        0       False   clear  22.14   
14957    fall     1      9    10    False        4        True   clear  23.78   
15407  winter     1     10     4    False        2        True    rain  14.76   
8391   spring     0     12     7    False        3        True   misty  14.76   

       feel_temp  humidity  windspeed  
3864      27.275      0.60    15.0013  
15798     25.760      0.83     7.0015  
11766     25.000      0.48    35.0008  
12411     25.760      0.68     0.0000  
14882     25.000      0.77     0.0000  
...          ...       ...        ...  
11844     31.820      0.45    27.9993  
15543     25.760      0.73    15.0013  
14957     27.275      0.53     6.0032  
15407     17.425      0.87    11.0014  
8391      18.180      0.87     7.0015  

[13903 rows x 12 columns]
<class 'pandas.core.frame.DataFrame'>
Traceback (most recent call last):
  File "/common/ketrong/tpotexp/env/lib/python3.10/site-packages/pandas/core/arrays/categorical.py", line 591, in astype
    new_cats = new_cats.astype(dtype=dtype, copy=copy)
ValueError: could not convert string to float: 'fall'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/common/ketrong/tpotexp/tpot2/ImputerExperiments/Locally_run_code/main3.py", line 166, in <module>
    main()
  File "/common/ketrong/tpotexp/tpot2/ImputerExperiments/Locally_run_code/main3.py", line 155, in main
    utils.loop_through_tasks(reg_experiments, regression_id_list, 
  File "/common/ketrong/tpotexp/tpot2/ImputerExperiments/Locally_run_code/utils.py", line 157, in loop_through_tasks
    X_train, y_train, X_test, y_test = load_task(base_save_folder=save_folder, task_id=taskid, r_or_c= r_or_c)
  File "/common/ketrong/tpotexp/tpot2/ImputerExperiments/Locally_run_code/utils.py", line 123, in load_task
    preprocessing_pipeline.fit(X_train)
  File "/common/ketrong/tpotexp/env/lib/python3.10/site-packages/sklearn/base.py", line 1473, in wrapper
    return fit_method(estimator, *args, **kwargs)
  File "/common/ketrong/tpotexp/env/lib/python3.10/site-packages/sklearn/pipeline.py", line 469, in fit
    Xt = self._fit(X, y, routed_params)
  File "/common/ketrong/tpotexp/env/lib/python3.10/site-packages/sklearn/pipeline.py", line 406, in _fit
    X, fitted_transformer = fit_transform_one_cached(
  File "/common/ketrong/tpotexp/env/lib/python3.10/site-packages/joblib/memory.py", line 312, in __call__
    return self.func(*args, **kwargs)
  File "/common/ketrong/tpotexp/env/lib/python3.10/site-packages/sklearn/pipeline.py", line 1310, in _fit_transform_one
    res = transformer.fit_transform(X, y, **params.get("fit_transform", {}))
  File "/common/ketrong/tpotexp/env/lib/python3.10/site-packages/sklearn/utils/_set_output.py", line 313, in wrapped
    data_to_wrap = f(self, X, *args, **kwargs)
  File "/common/ketrong/tpotexp/env/lib/python3.10/site-packages/sklearn/base.py", line 1098, in fit_transform
    return self.fit(X, **fit_params).transform(X)
  File "/common/ketrong/tpotexp/tpot2/tpot2/builtin_modules/imputer.py", line 114, in fit
    self.imputer.fit(X[self.columns_], y)
  File "/common/ketrong/tpotexp/env/lib/python3.10/site-packages/sklearn/base.py", line 1473, in wrapper
    return fit_method(estimator, *args, **kwargs)
  File "/common/ketrong/tpotexp/env/lib/python3.10/site-packages/sklearn/impute/_base.py", line 421, in fit
    X = self._validate_input(X, in_fit=True)
  File "/common/ketrong/tpotexp/env/lib/python3.10/site-packages/sklearn/impute/_base.py", line 350, in _validate_input
    raise ve
  File "/common/ketrong/tpotexp/env/lib/python3.10/site-packages/sklearn/impute/_base.py", line 332, in _validate_input
    X = self._validate_data(
  File "/common/ketrong/tpotexp/env/lib/python3.10/site-packages/sklearn/base.py", line 633, in _validate_data
    out = check_array(X, input_name="X", **check_params)
  File "/common/ketrong/tpotexp/env/lib/python3.10/site-packages/sklearn/utils/validation.py", line 929, in check_array
    array = array.astype(new_dtype)
  File "/common/ketrong/tpotexp/env/lib/python3.10/site-packages/pandas/core/generic.py", line 6643, in astype
    new_data = self._mgr.astype(dtype=dtype, copy=copy, errors=errors)
  File "/common/ketrong/tpotexp/env/lib/python3.10/site-packages/pandas/core/internals/managers.py", line 430, in astype
    return self.apply(
  File "/common/ketrong/tpotexp/env/lib/python3.10/site-packages/pandas/core/internals/managers.py", line 363, in apply
    applied = getattr(b, f)(**kwargs)
  File "/common/ketrong/tpotexp/env/lib/python3.10/site-packages/pandas/core/internals/blocks.py", line 758, in astype
    new_values = astype_array_safe(values, dtype, copy=copy, errors=errors)
  File "/common/ketrong/tpotexp/env/lib/python3.10/site-packages/pandas/core/dtypes/astype.py", line 237, in astype_array_safe
    new_values = astype_array(values, dtype, copy=copy)
  File "/common/ketrong/tpotexp/env/lib/python3.10/site-packages/pandas/core/dtypes/astype.py", line 179, in astype_array
    values = values.astype(dtype, copy=copy)
  File "/common/ketrong/tpotexp/env/lib/python3.10/site-packages/pandas/core/arrays/categorical.py", line 602, in astype
    raise ValueError(msg)
ValueError: Cannot cast object dtype to float64
srun: error: esplhpc-cp001: task 0: Exited with exit code 1
